alacritty_terminal/src/term/mod.rs

```diff
@@ -2184,6 +2193,28 @@ impl<T: EventListener> Handler for Term<T> {
     }
 }
 
+/// Terminal version for escape sequence reports.
+///
+/// This reports Alacritty's current version as a unique number based on its semver version. The
```

Comment:
This should be about `alacritty_terminal` crate version and not Alacritty in general, right?

Reply:
Incorrect, the crate version depends on the application `alacritty_terminal` is included in.

Reply:
I might word it strangely, but I mean that it has nothing to do with alacritty, other users likely has different names of their front end.

Reply:
I mean it's internal documentation anyways, not public for library consumers. For all intents and purposes, alacritty_terminals is also part of Alacritty.

Reply:
Well, I'd may be phrase it in a following way tbqh. What do you think?

```suggestion
/// This reports terminal's current version as a unique number based on its semver version. The
```

Reply:
I've considered it, on the other hand if the terminal doesn't use semver this might not actually report the correct thing at all.

I feel like in general making statements about Alacritty in `alacritty_terminal` shouldn't be prohibited, so I'm not sure how I feel about trying to be generic here. Especially with internal-only documentation like this.